### DISCUSSION TITLE
zh/guide/drivers/vtencent

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#298] [Announcements]
# zh/guide/drivers/vtencent

腾讯智能创作平台官网链接：https://app.v.tencent.com Cookie 随意在F12请求中找到一个携带Cookie的参数复制就可以 建议使用靠后刷新出来的请求中的Cookie参数，会携带含有 tf_x 参数的Cookie; login、`DescribeTasks、DescribeAccount、DescribeUserProfil...

https://alist.nn.ci/zh/guide/drivers/vtencent.html

**Comment by xxcyou3** [comment]
不能上传

**Reply by JayeDieyan** [reply]
不是不能上传，是只能上传媒体资源，图片视频这些，其它类型的从何端都上传不了

**Reply by endercatone** [reply]
一样，图片也上传不了

**Reply by JayeDieyan** [reply]
不能上传媒体资源的可以试试腾讯智能平台网页端，如果媒体资源包含违禁词或者平台禁止上传的资源同样会在上传时过滤掉。一般网页端能上传的一般alist及挂盘后都可上传。如果是在文件夹里，可以在网页端建好文件夹，在Alist建文件夹不一定会准确同步建立到对应位置。

**Comment by JayeDieyan** [comment]
只支持拉取50个文件列表

**Reply by xxcyou3** [reply]
这限制也太多了吧

**Reply by JayeDieyan** [reply]
3.30.0修复了

**Comment by goldsoft8888** [comment]
上传的视频不能在线播放，但是下载后可以播放，请修复

**Reply by my208928** [reply]
解码问题吧，有时候可以播放，有时候播放几秒就不行了

**Reply by sombermengling** [reply]
视频不能在线播放
